seacrowd/sea_datasets/shopee_reviews_tagalog/shopee_reviews_tagalog.py

```diff
@@ -0,0 +1,122 @@
+import csv
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+from datasets.download.download_manager import DownloadManager
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """
+@article{riegoenhancement,
+  title={Enhancement to Low-Resource Text Classification via Sequential Transfer Learning},
+  author={Riego, Neil Christian R and Villarba, Danny Bell and Sison, Ariel Antwaun Rolando C and Pineda, Fernandez C and Lagunzad, Herminino C}
+  journal={United International Journal for Research and Technology},
+  volume={4},
+  issue={8},
+  pages={72--82}
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["fil", "tgl"]
+_DATASETNAME = "shopee_reviews_tagalog"
+_DESCRIPTION = """\
+The Shopee reviews dataset is constructed by randomly taking 2100 training
+samples and 450 samples for testing and validation for each review star from 1
+to 5. In total, there are 10500 training samples and 2250 each in validation and
+testing samples.
+"""
+
+_HOMEPAGE = "https://huggingface.co/datasets/scaredmeow/shopee-reviews-tl-stars"
+_LICENSE = Licenses.MPL_2_0.value
+_URLS = {
+    "train": "https://huggingface.co/datasets/scaredmeow/shopee-reviews-tl-stars/resolve/main/train.csv",
+    "validation": "https://huggingface.co/datasets/scaredmeow/shopee-reviews-tl-stars/resolve/main/validation.csv",
+    "test": "https://huggingface.co/datasets/scaredmeow/shopee-reviews-tl-stars/resolve/main/test.csv",
+}
+
+_SUPPORTED_TASKS = [Tasks.SENTIMENT_ANALYSIS]
+_SOURCE_VERSION = "1.0.0"
+_SEACROWD_VERSION = "1.0.0"
+
+
+class ShopeeReviewsTagalogDataset(datasets.GeneratorBasedBuilder):
+    """Shopee Reviews Tagalog dataset from https://huggingface.co/datasets/scaredmeow/shopee-reviews-tl-stars"""
+
+    SOURCE_VERSION = datasets.Version(_SOURCE_VERSION)
+    SEACROWD_VERSION = datasets.Version(_SEACROWD_VERSION)
+
+    SEACROWD_SCHEMA_NAME = "text"
+    LABEL_CLASSES = ["0", "1", "2", "3", "4"]
```

Comment:
nit I think the labeling may not be very obvious, I would suggest to put an explanation (or maybe just make the label more verbose like "1 star", "2 stars").
```suggestion
    # "N" means N+1 star(s) review, e.g. "2" means 3 stars review
    LABEL_CLASSES = ["0", "1", "2", "3", "4"]
```